packages/dds/tree/src/test/feature-libraries/chunked-forest/chunkTree.spec.ts

```diff
@@ -136,12 +142,45 @@ describe("chunkTree", () => {
 				assert.equal(cursor.fieldIndex, 2);
 			}
 		});
+
+		it("encodes identifiers for in-memory representation", () => {
+			const identifierField: FieldKey = brand("identifierField");
+			const stringShape = new TreeShape(brand(stringSchema.identifier), true, [], true);
+			const identifierShape = new TreeShape(brand(JsonObject.identifier), false, [
+				[identifierField, stringShape, 1],
+			]);
+
+			const compressedId = testIdCompressor.generateCompressedId();
+			const stableId = testIdCompressor.decompress(compressedId);
+
+			const initialTree = {
+				type: brand(JsonObject.identifier),
+				fields: {
+					identifierField: [
+						{ type: brand("com.fluidframework.leaf.string"), value: stableId },
+					],
+				},
+			} satisfies JsonableTree;
```

Comment:
I don't think you need a type more specific than JsonableTree so just doing 
`const initialTree: JsonableTree =` instead of satisfies would probably be fine (and simpler) here.

That said, `singleJsonCursor({identifierField: stableId})` will give you an equivalent cursor, and is much shorter. JsonableTree is great since it can do any possible tree, but if you tree is valid in the JSON domain, there is a simpler option :)

Reply:
Updated to use `singleJsonCursor()` :)